dl/stubs.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build darwin
+// +build darwin
```

Comment:
Do we have to have a separate files `dl_darwin.go` and `stubs.go`?

Reply:
The main reason I separated them was to make it easier to add other OSs and architectures. Should I consolidate?

Reply:
I see. Let's keep separated then.